channeldb/channel.go

```diff
@@ -102,6 +102,11 @@ var (
 	// channel closure. This key should be accessed from within the
 	// sub-bucket of a target channel, identified by its channel point.
 	revocationLogBucket = []byte("revocation-log-key")
+
+	// frozenChanKey is the key where we store the information for any
+	// active "frozen" channels. This maps an outpoint to a "thaw height".
+	// This bucket is nested under the bucket for a specific channel.
```

Comment:
This is being used as a key, though the comment mentions it's a bucket.

Reply:
Dated comment, had it as a bucket at first. Fixed.